src/ebusd/mqtthandler.cpp

```diff
@@ -76,6 +78,7 @@ static vector<string> g_topicFields;
 static bool g_retain = false;             //!< whether to retail all topics
 static OutputFormat g_publishFormat = 0;  //!< the OutputFormat for publishing messages
 static bool g_ignoreInvalidParams = false;  //!< ignore invalid parameters during init
+static bool g_isMqttV311 = false;
```

Comment:
int g_mqttVersion

plus for all version related stuff:
#if ((LIBMOSQUITTO_MAJOR > 3) || ((LIBMOSQUITTO_MAJOR == 3) && ((LIBMOSQUITTO_MINOR > 1) || ((LIBMOSQUITTO_MINOR == 1) && (LIBMOSQUITTO_REVISION >= 1)))))
...
#endif